src/components/dialogs/parameters/dynamicsimulation/curve/dialog/equipment-filter.jsx

```diff
@@ -392,8 +377,11 @@ const EquipmentFilter = forwardRef(
                             virtualize
                             maxSelection={10}
                             options={voltageLevelIds}
-                            getOptionLabel={(value) => value}
-                            onChangeCallback={handleVoltageLevelChange}
+                            value={selectedVoltageLevelIds}
+                            getOptionLabel={(value) =>
+                                voltageLevelsMap.get(value) ?? value
+                            }
+                            onChange={handleVoltageLevelChange}
```

Comment:
Btw, this useCallback is useless no ? Why not using setSelectedVoltageLevelIds directly ?

Reply:
YES! I have removed debounce in the checkbox-autocomplete, do not need using useCallback.
I removed also some other overusing useCallback

DONE in https://github.com/gridsuite/gridstudy-app/pull/1836/commits/0bef9a2e0bd4c319f0e8f31aa7ea0cd139c91dc1

Reply:
Even beforethey were useless, a setter from useState does not change at everyrerender that's why eslint does not force us to put them in the deps array

Reply:
DONE